Lombiq.HelpfulLibraries.Cli/Lombiq.HelpfulLibraries.Cli.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Needs formatting, and license, Readme, icon references: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution#:~:text=Publish%20non%2Dsample%20projects%20as%20packages%20on%20NuGet.

Reply:
formatting?

Reply:
Indentation is inconsistent.